ferrocene/doc/user-manual/src/targets/index.rst

```diff
@@ -98,15 +98,15 @@ should not be used in production.
 
    * - :target:`x86_64-apple-darwin`
      - ``x86_64-apple-darwin``
-     - Host platform
+     - Cross-compilation
      - Full
-     - Also available as a cross-compile target for :target:`aarch64-apple-darwin`.
+     - \-
```

Comment:
We need to remove this then?

Reply:
I believe our aarch64 host can build x86_64 darwin targets.

Reply:
![image](https://github.com/ferrocene/ferrocene/assets/130903/1fd11e56-3f1e-42c4-9317-d775c8a8426b)
https://app.circleci.com/pipelines/github/ferrocene/ferrocene/3394/workflows/43ad5931-637b-41c6-baf5-f4ac9a5ac20f/jobs/16291?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link&utm_content=summary